setup.py

```diff
@@ -1153,6 +1178,12 @@ def main():
 
     install_requires += extra_install_requires
 
+    extras_require = {
+        "opt-einsum": ["opt-einsum>=3.3"],
+        "dynamo": ["jinja2"],
```

Comment:
Before we set "dynamo": ["jinja2"] only for linux builds now we do it for everything. Maybe we should keep it only for Linux ?

Reply:
Good catch, let me gate this with a check

Reply:
Fixed. As there is probably no rush to merge this right away.  Let me do one more round of binaries build here to confirm.